src/styleguide/_patterns/05-pages/00-page-layouts/01-error-page.json

```diff
@@ -0,0 +1,43 @@
+{
+  "title": "404",
```

Comment:
I would use `404 - Page not found`

Reply:
I changed that but then changed my mind as I thought most people visiting would be reasonable tech savvy

Reply:
I wouldn't expect that, given a good percentage of visitors are supposedly clients which might not be.